### PR TITLE
Adding everyTime mode

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/24365

### CODE REVIEW

**Comment by ChunxiAlexLuo** [comment]
@dockerymick I update some information and please free to polish the words.

**Review by ChunxiAlexLuo — CHANGES_REQUESTED** [review]
@dockerymick I update some information and please free to polish the words.

https://github.com/stolostron/rhacm-docs/pull/3931/commits/e3f42d97eea17a97e258802c6f3141b5bff72d45

File: governance/ansible_grc.adoc [hunk 1]
```diff
@@ -69,9 +70,12 @@ spec:
 
 . In the `extra_vars` section, add any parameters you need to pass to the Ansible job template.
 
-. Set the mode to either `once` or `disabled`.  The `once` mode runs the job one time and then the mode is set to `disabled`.
+. Set the mode to either `once`, `everyEvent` or `disabled`.
```
Comment (oafischer) [inline governance/ansible_grc.adoc]: Add comma after `everyEvent`

**Review by oafischer — APPROVED** [review]
Thanks Mikela, good catch on the special tag colon. Just left one small comment so I'm approving.

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Comment by dockerymick** [comment]
@oafischer @ChunxiAlexLuo Thanks to you both for your feedback, contributions, and approval. Merging this PR

**Pull request merged**